provider/kubernetes.go

```diff
@@ -160,12 +163,27 @@ func (provider *Kubernetes) loadIngresses(k8sClient k8s.Client) (*types.Configur
 					log.Warnf("Unknown value of %s for traefik.frontend.passHostHeader, falling back to %s", passHostHeaderAnnotation, PassHostHeader)
 				}
 
+				witelistSourceRangeString := i.Annotations[annotationTraeffikWhitelistSourceRange]
+				if len(witelistSourceRangeString) == 0 {
+					witelistSourceRangeString = i.Annotations[annotationKubernetesWhitelistSourceRange]
+				}
+
+				whitelistSourceRangeStrings := strings.Split(witelistSourceRangeString, ",")
+				var whitelistSourceRange []string
+				for _, s := range whitelistSourceRangeStrings {
+					s = strings.TrimSpace(s)
+					if len(s) > 0 {
+						whitelistSourceRange = append(whitelistSourceRange, s)
+					}
+				}
```

Comment:
We have this function elsewhere already. Can we outsource it into a central location? Maybe a utility package if we don't have one yet?